code/protected/models/Inventory.php

```diff
@@ -308,4 +308,165 @@ public function getPrevDayInv(){
         return $this->prev_day_inv;
     }
 
+    public function readInventoryUploadedFile($uploadedFile, $logFile, $w_id){
+        $connection = Yii::app()->secondaryDb;
+        $date = '';
+        $i=0;
+        $log =array();
+        $first = true;
+        $parentData = array();
+        $idMap = array();
+        while(!feof($uploadedFile)){
+            $row = fgetcsv($uploadedFile);
+            if(!$first && $row[0] != ''){
+                if(empty($date)){
+                    $date = $row[4];
+                }
+                $sql = 'select id from inventory where warehouse_id = '.$w_id.' and base_product_id = '.$row[0].' and date = "'.$row[4].'"';
+                $command = $connection->createCommand($sql);
+                $result = $command->queryScalar();
+                
+                if(array_key_exists($row[10], $parentData)){
+                    $parentData[$row[10]]['present_inv'] += $row[5];
+                    $parentData[$row[10]]['liquid_inv'] += $row[6];
```

Comment:
assign all row variable to named variables at one place, trim and initialize to default value